File: src/main/java/com/extendedae_plus/common/registry/block/assemblerMatrix/coreAdvancedCrafter/BlockAdvancedCrafter.java
package com.extendedae_plus.common.registry.block.assemblerMatrix.coreAdvancedCrafter;

import com.extendedae_plus.common.init.ModItems;
import com.glodblock.github.extendedae.common.blocks.matrix.BlockAssemblerMatrixBase;
import net.minecraft.world.item.Item;

public class BlockAdvancedCrafter extends BlockAssemblerMatrixBase<BlockEntityAdvancedCrafter> {
    @Override
    public Item getPresentItem() {
        return ModItems.CoreAdvancedCrafter.get();
    }
}

File: src/main/kotlin/com/extendedae_plus/common/registry/block/assemblerMatrix/coreAdvancedCrafter/BlockAdvancedCrafter.kt
package com.extendedae_plus.common.registry.block.assemblerMatrix.coreAdvancedCrafter

import com.extendedae_plus.common.init.EAEPItems
import com.fish.fishlib.util.extension.invoke
import com.glodblock.github.extendedae.common.blocks.matrix.BlockAssemblerMatrixBase
import net.minecraft.world.item.BlockItem

class BlockAdvancedCrafter : BlockAssemblerMatrixBase<TileAdvancedCrafter>() {
    override fun getPresentItem(): BlockItem = EAEPItems.CoreAdvancedCrafter()
}
